### ISSUE TITLE
[DOC] Change non-hardcoded references to "indices" to "indexes"

### ISSUE THREAD

**Issue by natebower** [opensearch-project/documentation-website#2358]
**What do you want to do?**
 
- [X] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
Please change non-hardcoded references to _indices_ to _indexes_.

 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
For example, the heading of [this page](https://opensearch.org/docs/latest/security-plugin/configuration/system-indices/) should be "System indexes", and the second sentence should read "Even if your user account has read permissions for all indexes..."

**Comment by Naarcha-AWS** [comment]
This is the same as #1046. I'm ok with closing that one and tracking against this if you would prefer.

**Comment by natebower** [comment]
Yeah, let's do that so that we're more current.

**Comment by hdhalter** [comment]
This one has been fixed: https://opensearch.org/docs/latest/security/configuration/system-indices/, but there are still others.

**Comment by BigSamu** [comment]
@Naarcha-AWS, can I take this issue?

**Comment by BigSamu** [comment]
> This one has been fixed: https://opensearch.org/docs/latest/security/configuration/system-indices/, but there are still others.

@hdhalter should be URLs also updated? In the quote you described thinking the URL should be [https://opensearch.org/docs/latest/security/configuration/system-indixes/](https://opensearch.org/docs/latest/security/configuration/system-indixes/) instead of [https://opensearch.org/docs/latest/security/configuration/system-indices/](https://opensearch.org/docs/latest/security/configuration/system-indices/). Just to keep consistency.

**Comment by hdhalter** [comment]
Hi @BigSamu - Thanks so much for taking this on! I didn't change the URL at the time because changing the URL has a ripple effect, that is, you have to make sure you're not introducing any broken links. You also have to add a redirect in the file. Let me know if you need more info.

**Comment by BigSamu** [comment]
Hey @hdhalter, let me take a look at it. I think I can do the proper updates without braking anything.

Regards,

Samuel

**Comment by BigSamu** [comment]
@Naarcha-AWS @hdhalter,

I just finished opening PR #5353 for this issue. After further inspection, the change of non-harcoded variables is going to be tricky and needs a planning with the owners of each part of the source code, I think.

The PR created in this case is replacing the references of 'indices' to 'indexes' ONLY in the cases of description texts. Those changes were NOT executed for the cases where the 'indices' term was used as variables, in links, as properties, or another part in the code base.

I would appreciate your feedback to see how I can continue helping on this issue. Maybe it will be useful to open a set of individual issues, bringing the owners of each part of the source code to work together with the documentation team, so the update for the code and the documentation could be done in parallel at the same time.

Also, I suggest to add may be a banner in the documentation website detailing that OpenSearch is in the process of changing 'indices' references to 'indexes'. That will avoid confusions I believe.

Regards,

Samuel

**Comment by Naarcha-AWS** [comment]
Closing this issue for now. We can start an initiative to update the other plugins in a separate issue.

**Comment by BigSamu** [comment]
> Closing this issue for now. We can start an initiative to update the other plugins in a separate issue.

@Naarcha-AWS count with me for that initiative.
